galileo/opt/include/galileo/opt/TrajectoryOpt.h

```diff
@@ -331,12 +392,12 @@ namespace galileo
             std::vector<std::shared_ptr<ConstraintData>> G;
             std::shared_ptr<DecisionData> Wdata = std::make_shared<DecisionData>();
 
-            auto num_phases = sequence->getNumPhases();
+            size_t num_phases = sequence->getNumPhases();
 
             printf("Starting initialization\n");
             auto begin_time = std::chrono::high_resolution_clock::now();
             auto start_time = std::chrono::high_resolution_clock::now();
-            auto end_time = std::chrono::high_resolution_clock::now();  
+            auto end_time = std::chrono::high_resolution_clock::now();
```

Comment:
I would remove the timing, and the dependency on chrono

It's not needed in  the actual evaluation. 

Possibly add an "enable timing" option, if this is really needed for testing/benchmarking